run_tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+TEMP=$(date +%s)
+
+docker build -t supervisor-tests:$TEMP -f Dockerfile.tests --build-arg CACHEBUST=$TEMP .
+
```

Comment:
Building the image here and deleting it below?

Reply:
Can we have something useful in between however minimalistic possible?

Reply:
@anmolbabu Useful, as in? I think a lot of steps will get stuffed in gradually as we move ahead.

Reply:
probably run the supervisor-tests container and collect success or failure(if possible?) at this moment?

Reply:
I understand that build time itself, tests would be run but if you see https://github.com/fabric8-analytics/fabric8-analytics-server/blob/master/runtest.sh#L72 I feel its for some specific reason related to observing post deployment conditions. Now, if its just to get db setup and run tests against it I am ok with not having it but please check the purpose of it with @miteshvp